userguide/virtualmachines.rst

```diff
@@ -156,89 +156,77 @@ a Virtual Machine (VM) type are described in
    +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
    | 4        | MAC Address        | string         | Enter the desired MAC address to override the auto-generated                                  |
    |          |                    |                | randomized MAC address.                                                                       |
-   |          |                    |                |                                                                                               |
    +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
    | 4        | Attach NIC         | drop-down menu | Select the physical interface to associate with the VM.                                       |
-   |          |                    |                |                                                                                               |
-   |          |                    |                |                                                                                               |
    +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
-   | 5        | Optional: Choose   | browse button  | Click |ui-browse| to select an installer ISO or image file on the %brand% system.             |
-   |          | installation media |                |                                                                                               |
+   | 5        | Optional: Choose   | browse button  | Click |ui-browse| to select an installer ISO or image file on the %brand%                     |
+   |          | installation media |                | system.                                                                                       |
    |          | image              |                |                                                                                               |
    +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
    | 5        | Upload ISO         | checkbox and   | Set to upload an installer ISO or image file to the %brand% system.                           |
-   |          |                    | buttons        |                                                                                               |
    +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
 
 
 The final screen of the Wizard displays the chosen options for the new
 Virtual Machine (VM) type. Click :guilabel:`SUBMIT` to create the VM or
-:guilabel:`BACK` to change any settings.
+:guilabel:`BACK` to change any settings. After the media has been
```

Comment:
Better!  Removing the media is a change of subject, so should be in a new paragraph:
```suggestion
:guilabel:`BACK` to change any settings.

After the VM has been installed, remove the install media
```